runtime/commonTest/src/kotlinx/serialization/modules/ModuleBuildersTest.kt

```diff
@@ -253,14 +255,14 @@ class ModuleBuildersTest {
         val m1 = serializersModuleOf(Unit::class, delegate)
         val m2 = serializersModuleOf(Unit::class, delegate2)
         val aggregate = m1 + m2
-        assertEquals(delegate2, aggregate.getContextual<Unit>())
-        assertEquals(delegate, aggregate.getContextual<Unit>())
+        assertEquals(delegate2, aggregate.getContextual(Unit::class))
+        assertEquals(delegate, aggregate.getContextual(Unit::class))
     }
 
     @Test
     fun testThrowOnTheSamePolymorphicSerializer() {
-        val m1 = SerializersModule { polymorphic<Any> { A::class with A.serializer() } }
-        val m2 = SerializersModule { polymorphic<Any> { A::class with ASerializer } }
+        val m1 = SerializersModule { polymorphic<Any>(Any::class) { subclass(A.serializer()) } }
```

Comment:
is `<Any>` really needed given we now have `Any::class`?

Reply:
No, it's just an automatic migration